src/main/java/DanChat.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Scanner;
+
+public class DanChat {
+    public static void printMessage(String message) {
+        System.out.println("=============================");
+        System.out.println("\t" + message);
+        System.out.println("=============================");
+    }
+    public static void printTaskList(Task[] tasks) {
+        int taskNumber = 1; //Output index for list items, starting from 1.
+        for (Task task: tasks) {
+            if (task == null) {
+                break;
+            }
+            String taskName = task.getDescription();
+            String taskStatusIcon = task.getStatusIcon();
+            System.out.println("\t" + taskNumber + ". " + "[" + taskStatusIcon + "] " + taskName);
+            taskNumber++;
+        }
+    }
+    public static boolean isValidInteger(String str) {
+        try {
+            Integer.parseInt(str);
+            return true;
+        } catch (NumberFormatException e) {
+            return false;
+        }
+    }
+    public static void main(String[] args) {
+        String line = "=============================";
```

Comment:
Variables that are not meant to be changed during runtime should be changed to constants by declaring them with `final`.